### DISCUSSION TITLE
cannot permanently remove agent

### DISCUSSION THREAD

**Discussion by Ten-Gu** [Ylianst/MeshCentral#6303] [Issues]
Hi, I somehow can't manage to uninstall an agent.
I have tested the following:
on the client
/usr/local/mesh_services/meshagent/meshagent -fulluninstall
on the server (Web - console tab): uninstallAgent
on the server (web): group action - uninstall agent

When I restart the client, it reappears.
How do I get rid of the agent completely?

**Comment by si458** [comment]
One thing to check is the location of the service,
So type `info` in the console tab of the device and it will show you its application location

But also if I remember it's `-fullinstall` for Windows and `-uninstall` for linux, will double check soon
